tests/clickhouse-test

```diff
@@ -1019,6 +1022,12 @@ def main(args):
                 else:
                     parallel_tests.append(test)
 
+            j2env = jinja2.Environment(
+                loader=jinja2.FileSystemLoader(suite_dir),
+                keep_trailing_newline=True,
+            ) if USE_JINJA else None
+            j2env.globals['uuid'] = lambda: uuid.uuid4().hex
```

Comment:
What is the use-case for this function? Do we want to have random unique identifier at each run?

Reply:
Right now - no. Ok, let's simply remove it.